kuma/javascript/src/utils.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Helper function to get cookie value given a cookie name
+ * @params {string} cookie name
+ * @returns {string} cookie value
+ */
+export function getCookie(name) {
+    var cookieValue = null;
+    if (document.cookie && document.cookie !== '') {
+        var cookies = document.cookie.split(';');
+        for (var i = 0, l = cookies.length; i < l; i++) {
+            var cookie = cookies[i].trim();
```

Comment:
Nit (and again I realize I didn't catch this in https://github.com/mozilla/kuma/pull/5469): I think these first two lines could be simplified to:
```javascript
for (var cookie of cookies) {
    cookie = cookie.trim()
    ...
}
```